src/viur/core/bones/base.py

```diff
@@ -319,6 +319,10 @@ def __init__(
 
         self.compute = compute
 
+    @property
+    def descr(self):
+        return self._descr or self.name or ""
```

Comment:
Can you merge the main --> develop first? I miss https://github.com/viur-framework/viur-core/pull/1223


Then we could think about

```suggestion
        return self._descr or translate(self.name) or ""
```

Reply:
ok will do.

Reply:
I've fixed it differently, see current diff.